bpf/bpf_host.c

```diff
@@ -828,7 +821,16 @@ do_netdev(struct __ctx_buff *ctx, __u16 proto, const bool from_host)
 #endif
 #ifdef ENABLE_IPV4
 	case bpf_htons(ETH_P_IP):
-		identity = resolve_srcid_ipv4(ctx, identity, &ipcache_srcid,
+		/* This is the first time revalidate_data() is going to be called in
```

Comment:
Could you potentially drop this chunk all together? Looks like you nullify it in the next commit anyway

Reply:
You mean like shuffling the second and third patch around? Yeah let's try :thinking:.

Reply:
yes, looks like you add this chunk, but then delete it in the third commit anyway. 

Anyway, its a nit, is just to make the history a bit cleaner.